CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v8.1.0
+------------------------------
+*October 9, 2018*
```

Comment:
Very minor, but if you change this to today's date I'll be happy to approve!

Reply:
✅